### PR TITLE
Help card first go!

### PR DESCRIPTION
**Pull request by HSalmon**
screen shot 2015-08-28 at 4 51 28 pm

### CODE REVIEW

File: lineman/app/components/group_page/group_help_card/group_help_card.haml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+%section.group-help-card
+  %h2.lmo-card-heading Help
+  <iframe width="269" height="150" src="https://www.youtube.com/embed/B1gLEAKPkOc" frameborder="0" allowfullscreen></iframe>
+  .group-help-card__articles
+    .media
+      .media-left
+        %img.group-help-card__article-thumbnail{src: '/img/efficient-meetings-thumbnail.png', alt: 'Weaving in agenda points'}
+      .media-body
+        %a.group-help-card__article-title{href: 'http://blog.loomio.org/2015/07/16/more-efficient-in-person-meetings-with-loomio/', title: 'More efficient in person meetings with loomio'}
+          %strong More efficient in-person meetings with Loomio
```
Comment (gdpelican) [inline lineman/app/components/group_page/group_help_card/group_help_card.haml]: Translations?

**Comment by rdbartlett** [comment]
Rob mentioned that localising this is going to be a bit tricky. I didn't get any of the context though, so keen to hear more. 

I'm happy for the first deploy to be English-only, but yes we will want to resolve that soon.

I'm assuming that soon this content will be hosted on something like help.loomio.org/en/[:id]/getting-started-with-loomio so we can easily switch over to help.loomio.org/es/[:id]/empezar-con-loomio

But I haven't given it much thought yet.

**Comment by robguthrie** [comment]
We knew that this was not going to be production ready but it helped us understand what is involved in this type of thing.

Conclusions we came to were that first pass should be EN only and hardcoded content. Later on we can move to RSS feed syndication, IE: this is just a blog RSS widget for our help site. If that site has multiple language streams that'll be great.

**Comment by gdpelican** [comment]
Closing this for now; looking forward to the next iteration!

**Pull request closed without merging**